docs/tools/aiven-console/howto/create-accounts.rst

```diff
@@ -1,23 +1,49 @@
-Create accounts
-===============
+Create organizations and organizational units
+==============================================
 
-**Accounts** are groups of projects with common settings like authentication and teams (user groups).
+**Organizations** and **organizational units** (or **unit**) can be used to group projects and apply common settings like authentication and teams (user groups). For details and recommendations on creating hierarchical organizations in Aiven, see :doc:`Organizations, projects, and managing access permissions </docs/platform/concepts/projects_accounts_access>`.
 
-Create a new account
-------------------------------
 
-#. At the top right side of the page, click the account name and select **Create account**. 
+Create an organization
```

Comment:
We should add here that we don't recommend more than one organization per customer but rather the usage of Units. We should also mention here what all is managed per organization.

- Billing Groups
- Authentication
- Team

Another org would mean all this needs to be resetup.